src/common/ceph_context.cc

```diff
@@ -285,11 +294,11 @@ class CephContextObs : public md_config_obs_t {
                           const std::set <std::string> &changed) override {
     if (changed.count(
 	  "enable_experimental_unrecoverable_data_corrupting_features")) {
-      ceph_spin_lock(&cct->_feature_lock);
+      ceph::spin_lock(&cct->_feature_lock);
       get_str_set(
```

Comment:
As with std::lock_guard, I'm absolutely happy to do it-- but it may be a better "next" PR than this one (or, alternatively, it may be perfect for this one). I guess the question is whether the cycles are better spent here, or on trying to fix things for Luminous.